.circleci/config.yml

```diff
@@ -55,10 +54,10 @@ jobs:
             echo 'export NVM_DIR="/opt/circleci/.nvm"' >> $BASH_ENV
             echo "[ -s \"$NVM_DIR/nvm.sh\" ] && . \"$NVM_DIR/nvm.sh\"" >> $BASH_ENV
       - node/install-yarn
-      - cypress/install:
-          cache-key: cache-{{ arch }}-{{ .Branch }}-{{ checksum "tdrs-frontend/package.json" }}
-          working_directory: tdrs-frontend
-          yarn: true
+      - node/install-packages:
+          app-dir: tdrs-frontend
+          cache-path: ~/.cache
```

Comment:
This usage of the `cache-path` variable was the key difference that made this work where it didn't before. By pointing to the `~/.cache` directory instead of the default (`node_modules`) we capture both the Yarn registry and the Cypress binary in the same cache, which will be applied to subsequent builds.

Reply:
appreciate the explanation here! quick q for you @jtwillis92--TTA/HS  team shared a link with us to their config for the owasp report (https://github.com/HHS/Head-Start-TTADP/blob/main/.circleci/config.yml#L280-L298). does anything in here standout to provide any extra reassurance that our fix will work when we merge into `hhs:main`?

Reply:
Yes, looks like they use the same `node` orb we're using here so we shouldn't run into any issues.